datasource/timeouts/schema_test.go

```diff
@@ -58,6 +124,74 @@ func TestBlock(t *testing.T) {
 	}
 }
 
+func TestAttributesWithOpts(t *testing.T) {
+	t.Parallel()
+
+	type testCase struct {
+		opts     timeouts.Opts
+		expected schema.Attribute
+	}
+	tests := map[string]testCase{
+		"empty-opts": {
+			opts: timeouts.Opts{},
+			expected: schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"read": schema.StringAttribute{
+						Optional: true,
+						Validators: []validator.String{
+							validators.TimeDuration(),
+						},
+					},
```

Comment:
Similarly here regarding the default description. I wonder if I need to double check the `Equal()` implementation.